### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by GH6** [superserrano/7DaysLocalizer#1]
When I pasted the secret api key and tried to translate the localization file, the file with 1kb capacity was saved.
The original Localization file has key,english and 170kb files, while the resulting Localization file is
Key,File,Type,UsedInMainMenu,NoTranslate,english,Context / Alternate Text,german,spanish,french,italian,japanese,koreana,polish,brazilian,russian,turkish,schinese,tchinese

That's all it says.
Is there anything else I need to do? Whatever translation model you use, the results are the same.

log:
Initializing form...
Opening original file...
C:\Age of Oblivion\00-CoreFiles-AOO\Config\Localization.txt
Parsing original file...
Index was outside the bounds of the array.
Original file parsed and ready.
Initializing OpenAI...
OpenAI Initialized.
Beginning translation
File Translated.
Saving translation to file...
File saved to C:\Age of Oblivion\00-CoreFiles-AOO\Localization.txt
Opening original file...
Parsing original file...
Empty path name is not legal.
Original file parsed and ready.
Initializing OpenAI...
OpenAI Initialized.

**Comment by as1472580** [comment]
Yeah, I've run into this situation too. Not sure if it's something I'm doing wrong.

**Comment by K1F3R** [comment]
did you manage to solve the problem?

**Comment by GH6** [comment]
No, it hasn't been resolved. To be precise, I haven't paid attention to this issue since posting it on GitHub. I've just been manually copying and pasting or asking ChatGPT-4o for help.

> did you manage to solve the problem?
